src/aux-runtime/runtime/AuxLibrary.spec.ts

```diff
@@ -17069,6 +17085,26 @@ describe('AuxLibrary', () => {
                 'data:image/png;base64,AQIDBAU='
             );
         });
+        it('should convert the given value to bytes when given an ArrayBuffer', () => {
```

Comment:
@TroyceGowdy This test name is wrong. It should explain that the function should convert the value to a base64 string.